### PR TITLE
[WIP] v1.11 - Updates quickstarts/tutorials/pub-sub to dotnet v7

### PR DESCRIPTION
**Pull request by robertojrojas**
Signed-off-by: Roberto J Rojas <robertojrojas@gmail.com>

# Description

Updates quickstarts/tutorials/pub-sub/csharp-subscriber to use dotnet v7.

### CODE REVIEW

File: tutorials/pub-sub/csharp-subscriber/csharp-subscriber.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
```
Comment (artursouza) [inline tutorials/pub-sub/csharp-subscriber/csharp-subscriber.csproj:4]: Does it also need to change the Docker image?

**Comment by greenie-msft** [comment]
@robertojrojas thanks for the contribution. Can you clarify why you only updated the pubsub tutorial and not the other quickstarts for using .net7?

**Review by artursouza — APPROVED** [review]

**Comment by artursouza** [comment]
I think we should update all quickstarts to dotNet 7

**Comment by robertojrojas** [comment]
@greenie-msft  I noticed that the [distributed-calculator/csharp app](https://github.com/dapr/quickstarts/blob/cb4463f1a19d51253f539aa0b40c66bb3e0beaa4/tutorials/distributed-calculator/csharp/Subtract.csproj#L4) was updated to use the v7 version and this was a simple change to make the tutorials "consistent" with other tutorials using the dotNet framework.  I do believe the rest of the quickstarts need to be updated, but maybe for release v1.11+.

**Comment by robertojrojas** [comment]
Actually, so we don't delay the v1.10 release with this PR, I'll postpone this change for v1.11.

**Review by paulyuk — APPROVED** [review]
thank you for this contribution.  lgtm.

**Pull request merged**